src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -141,17 +142,29 @@ function MoneyRequestConfirmPage(props) {
                 return;
             }
 
-            IOU.requestMoney(
-                props.report,
-                props.iou.amount,
-                props.iou.currency,
-                props.currentUserPersonalDetails.login,
-                props.currentUserPersonalDetails.accountID,
-                selectedParticipants[0],
-                trimmedComment,
-            );
+            FileUtils.readFileAsync(props.iou.receiptPath, props.iou.receiptSource).then((receipt) => {
```

Comment:
I am assuming scan receipt is optional, so why do we call `FileUtils.readFileAsync` everytime? Souldn't we call this only if we have a receiptPath or similar check to confirm this request has a receipt?

Reply:
Yea, makes sense. I pushed changes for this.